lib/api/v1/user-controller.js

```diff
@@ -34,24 +34,28 @@ module.exports = function(app, user) {
   app.use(express.json());
 
   // Get a list of all Users
-  app.get('/api/v1/user', user.can('manage users'), function(req, res) {
-    User.find({}, '-password', function(err, users) {
+  app.get('/api/v1/user', user.can('view users'), function(req, res) {
+    var query = {};
+    if (req.user && req.user.role === 'viewer')
+      query.username= req.user.username;
```

Comment:
This is really a bit of a hack. Instead of checking that the role is `viewer`, we should really be checking that the user does not have the `admin users` permission.